src/seedu/addressbook/data/tag/Tag.java

```diff
@@ -50,4 +50,5 @@ public String toString() {
         return '[' + tagName + ']';
     }
 
+    public String getRawString() { return tagName; }
```

Comment:
Please refer to the coding standards, and see how this method should be formatted :)